rest_api/sawtooth_rest_api/route_handlers.py

```diff
@@ -88,13 +90,12 @@ def batches_post(self, request):
             data = None
             link = link.replace('batch_status', 'batches')
 
-        return RouteHandler._wrap_response(
+        return self._wrap_response(
             data=data,
             metadata={'link': link},
             status=status)
 
-    @asyncio.coroutine
-    def status_list(self, request):
+    async def list_statuses(self, request):
```

Comment:
Does this change require a certain level of Python 3? (3.5, I think?). If we're fine with that constraint, we will need to be sure that we're documenting the minimum python requirements. if not, we can't make this change.